services/commands-service.ts

```diff
@@ -62,6 +73,11 @@ export class CommandsService implements ICommandsService {
 				} else {
 					command.execute(commandArguments).wait();
 				}
+				
+				if (this.commandHelp[commandName]) {
```

Comment:
the contents of `this.$staticConfig.COMMAND_HELP_FILE_NAME` are read as JSON and `commandHelp` is defined as a `get` property so it is basically a dictionary